proposals/restructure-KF-docs-proposal.md

```diff
@@ -0,0 +1,232 @@
+This is an initial proposal, to be iterated on, following the discussion in [#2293](https://github.com/kubeflow/website/issues/2293).
+
+### Kubeflow documentation structure
+
+- **Getting started**
+  - Introduction (current About/Kubeflow)
+  - Concepts (current Kubeflow Overview)
+  - Use Cases (current About/Use Cases)
+  - Install Kubeflow (current `Installing Kubeflow`, to redirect to `Deployment` homepage)
+  - [delete] Cloud Installation
+    
+- **Community**
+  - About (current About/Community)
+  - Contributing (current About/Contributing to Kubeflow + details on Working Groups)
+  - Documentation (merge current About/Docs + About/Style Guide for Kubeflow docs)
+  - Events (current About/Events Calendar)
+  - Google Summer of Code (current About/Google Summer of Code)
+    
+- **Components** (remove "of Kubeflow")
+  - Jupyter Notebooks (moved to under components)
+    - Overview of Jupyter Notebooks in Kubeflow
+    - Set Up Your Notebooks
+    - Create a Custom Jupyter Image
+    - Submit Kubernetes Resources
+    - Build a Docker Image on GCP
+    - Troubleshooting Guide
+  - Central Dashboard
+    - Central Dashboard 
+    - Registration Flow
+  - Metadata
+  - Fairing 
+    - Overview of Kubeflow Fairing
+    - Install Kubeflow Fairing
+    - Configure Kubeflow Fairing
+    - Fairing on Azure
+    - Fairing on GCP
+    - Tutorials
+    - Reference
+  - Feature Store
+    - Introduction to Feast
+    - Getting started with Feast
+  - Frameworks for Training
+    - Chainer Training
+    - MPI Training
+    - MXNet Training
+    - PyTorch Training
+    - TensorFlow Training (TFJob)
+  - Hyperparameter Tuning
+    - Introduction to Katib
+    - Getting started with Katib
+    - Running an experiment
+    - Katib Configuration Overview
+    - Environment Variables for Katib Components
+  - Pipelines
+    - Pipelines
+  - Tools for Serving
+    - Overview
+    - KFServing
+    - Seldon Core Serving
+    - BentoML
+    - NVIDIA Triton Inference Server
+    - TensorFlow Serving
+    - TensorFlow Batch Prediction
+  - Multi-Tenancy in Kubeflow
+    - Introduction to Multi-user Isolation
+    - Design for Multi-user Isolation
+    - Getting Started with Multi-user Isolation
+  - Miscellaneous
+    - Nuclio functions
+    
+- **Kubeflow Pipelines** (add "Kubeflow")
```

Comment:
Is there a rationale for having this a separate top-level item instead of just nesting it as a component? Is this because KFP can be installed standalone too?